acceptance/test.toml

```diff
@@ -0,0 +1 @@
+# root configuration applicable to all tests
```

Comment:
Are settings here intended to be inherited by configuration files in subdirectories?

Good to include as a comment if this is the case.

Reply:
Yes, inherited but as a whole, not merged. Updated the comment.